cdk/util.ts

```diff
@@ -44,5 +44,5 @@ export function lowerCaseEnvironmentId(id: string): string {
  * @returns true if the input is a non-empty string and false otherwise
  */
 export function isString(str: unknown): str is string {
-  return str && typeof str === "string";
+  return typeof str === "string" && !!str;
```

Comment:
Context is provided in the commit message. But this coerces the return type to `string` rather than `string | boolean`. The `&&` operator does not coerce types to boolean.

Reply:
Cool! Had to look that up. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_NOT#double_not_!!